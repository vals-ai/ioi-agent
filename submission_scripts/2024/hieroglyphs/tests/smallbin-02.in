version https://git-lfs.github.com/spec/v1
oid sha256:60c6a6a4f061ea6c6605980ec8245e681c7836581dbf51e1c51483413263bd42
size 57
