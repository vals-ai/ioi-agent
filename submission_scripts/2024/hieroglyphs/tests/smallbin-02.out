version https://git-lfs.github.com/spec/v1
oid sha256:8eb44c8555646a274fb825444e22860f29197c271332ee684f35bf9b97c219b7
size 46
